tests/rptest/e2e_tests/flink_scale_test.py

```diff
@@ -82,54 +88,140 @@ def assert_failed_jobs(job_list):
                 "Flink reports fails for " \
                 f"transaction workload:\n{desc}"
 
-        # TODO: Validate/Gather info on
-        # - Transactions can be generated per one flink job
-        # - Transactions per node
-        # - Transactions total >500K
+        def get_hwm_for_topic(topic):
+            _hwm = 0
+            for partition in self.rpk.describe_topic(topic):
+                # Add currect high watermark for topic
+                _hwm += partition.high_watermark
+            return _hwm
+
+        # Validate test according to
+        # - How many transactions can be generated per one job
+        # - How mane transactions for the node
 
         # Prepare topics
-        self.topic_name = "flink_scale_1"
-        self.topic_specs = [TopicSpec(name=self.topic_name, partition_count=8)]
-        for spec in self.topic_specs:
+        topics_per_node = 4
```

Comment:
what is the "per_node" part? (what node you are talking about) can't we just say num_topics or something.

Reply:
node here means flink node and how much workloads it will run per one node. Later on it was renamed to total_workloads